### ISSUE TITLE
No highlighting on players ready window

### ISSUE THREAD

**Issue by eric1294** [ValveSoftware/Dota2-Gameplay#16147]
### Description

After accepting a match, none of the squares light up.

### Screenshots

![bug](https://github.com/ValveSoftware/Dota2-Gameplay/assets/135648777/63ce4702-70b6-45c6-8a2e-49840e176388)

**Comment by HusrFX** [comment]
yes, i have same bug after last update. same. match id 7659583697

I am was sniper in that game. I decided to move on after this god, but to my surprise I didn’t see a button to re-enter the game and as a result, after half an hour, 24 rating points were taken from me, although we were leading in the game... just give me back the points that were taken away because of some small update. Well, or at least carry out a massive recalibration so that you don’t have to return the glasses manually
